### ISSUE TITLE
optional chart fill

### ISSUE THREAD

**Issue by hercek** [OpenTracksApp/OpenTracks#1823]
**Is your feature request related to a problem? Please describe.**
Charts can be very hard to parse when there is more data series displayed (e.g. 3 and more). 

**Describe the solution you'd like**
The most significant reason for this is the chart fill (alpha blended lighter fill below a chart curve). It decreases legibility **a lot** when there are many data series in the chart. Add an option to the settings which will allow users to switch off the chart fill.

**Describe alternatives you've considered**
Remove the chart fill completely from the code. I prefer this more but the chart fill increases chart legibility **slightly** when there is only one or maybe two data series shown. This option would keep settings cleaner.

**Additional context**
I will likely provide a PR for the proposed solution or the alternative solution if one of them is approved.

**Comment by dennisguse** [comment]
:+1: 
Can you share a screenshot how it looks like?

**Comment by hercek** [comment]
Ok, here are some samples. The older version does not contain also the accepted PR which moves x-axis label lower right. So the comparison is not 100% fair. But I do not have only one commit older APK ready.

**Portrait mode with chart fill**
![PortraitWithFill](https://github.com/OpenTracksApp/OpenTracks/assets/5325126/205b6561-2538-4bf3-aabd-2bae6cd72462)

**Portrait mode without chart fill**
![PortraitNoFill](https://github.com/OpenTracksApp/OpenTracks/assets/5325126/b532c358-36ef-4028-b14b-6903be82764f)

**Landscape mode with chart fill**
![LandscapeWithFill](https://github.com/OpenTracksApp/OpenTracks/assets/5325126/fd038c68-82c0-4597-9a1f-50572ea48938)

**Landscape mode without chart fill**
![LandscapeNoFill](https://github.com/OpenTracksApp/OpenTracks/assets/5325126/cab8c2d7-dba0-464a-871d-423217cda61c)

I think the problem is best visible in the landscape mode: try to follow the power curve in the interval from 4.2 km to 4.5 km (around the 3rd pin). It is more or less impossible in the version with the chart fill. Or I'm pretty much blind in my advanced age :)

Please specify whether you want version controlled by an option or just removing the fill all the time. Maybe it could be done with a 3-state checkbox - the 3-rd state would use fill for charts with only one or two data series.

**Comment by hercek** [comment]
Yes. I'll make a PR for this version: hiding fill when there are at least 3 data series visible.
